### DISCUSSION TITLE
Lag while loading playlists, songs preload, or on a single song also while loading.

### DISCUSSION THREAD

**Discussion by malikmajai** [Raptor123471/DingoLingo#91] [General]
Hi, 

I am using DingoLingo but I felt in multiple servers, it sometimes lag while loading songs. Is it due to my poor hosting server or everyone facing same issue?

**Comment by RafaelSolVargas** [comment]
How long do you consider as lag?
I tested DingoLingo now and it takes something like 2-3 seconds to load songs and playlists

**Reply by malikmajai** [reply]
Yeah, It lags 6-7 seconds on playlists. And when bot is online in multiple servers, multiple lag happens. I am just curious for the reason behind this lag. Because while all other commands, or when songs are not loading, music is playing perfectly without any lags. But just when somewhere bot loading songs, it lags in all other servers.
